### PR TITLE
update sanity-test with link to gallery test

### PR DESCRIPTION
**Pull request by jbinda**
Quick update to link proper test-cases for Gallery in sanity-test md

### CODE REVIEW

File: test-suites/gutenberg/sanity-tests.md [hunk 1]
```diff
@@ -1,18 +1,23 @@
...
 
-Gallery-2
+Gallery-2  
 
-- [ ] Gallery block - Insert image from device (failing) - [steps](https://github.com/wordpress-mobile/test-cases/blob/master/test-cases/gutenberg/gallery.md#tc006)
-- [ ] Gallery block - Insert image from device (cancel) - [steps](https://github.com/wordpress-mobile/test-cases/blob/master/test-cases/gutenberg/gallery.md#tc011)
+- [ ] Gallery block - Choose from device (stay in editor) - Successful upload- [steps](https://github.com/wordpress-mobile/test-cases/blob/master/test-cases/gutenberg/gallery.md#tc005)
+- [ ] Gallery block - Choose from device (stay in editor) - Failed upload - [steps](https://github.com/wordpress-mobile/test-cases/blob/master/test-cases/gutenberg/gallery.md#tc006)
+- [ ] Gallery block - Choose from device (leave the editor) - Successful upload - [steps](https://github.com/wordpress-mobile/test-cases/blob/master/test-cases/gutenberg/gallery.md#tc007)
+- [ ] Gallery block - Choose from device (close and re-open the editor with ongoing uploads) - [steps](https://github.com/wordpress-mobile/test-cases/blob/master/test-cases/gutenberg/gallery.md#tc008)
```
Comment (pinarol) [inline test-suites/gutenberg/sanity-tests.md]: Some test cases are duplicated. @mkevins as far as I see you added new cases [here](https://github.com/wordpress-mobile/test-cases/blob/master/test-cases/gutenberg/gallery.md)  
But these look same:
#tc001 == #tc008
#tc002 == #tc007
Could you remove duplicates and then update sanity-tests.md file accordingly? Thanks! YI think you can open a separate PR and we can close this one.

Comment (mkevins) [reply test-suites/gutenberg/sanity-tests.md:11]: :+1: I've deduplicated the test cases and updated the references for those tests in this PR: https://github.com/wordpress-mobile/test-cases/pull/21.

Comment (jbinda) [reply test-suites/gutenberg/sanity-tests.md:11]: ok so I can close this PR ?

**Comment by pinarol** [comment]
Closing this one in favor of https://github.com/wordpress-mobile/test-cases/pull/21

**Pull request closed without merging**